data/mods/Magiclysm/items/tools.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "id": "cauldron_demon_chitin",
+    "type": "TOOL",
+    "category": "container",
+    "name": "cauldron of poison",
+    "description": "This cauldron made of demon spider chitin seems to absorb the light.  It will hold 16 liters of material and will absorb poisons from it.  It may have other properties that require discovery.",
+    "weight": "1424 g",
+    "volume": "20 L",
+    "price": 2000,
+    "to_hit": -1,
+    "looks_like": "clay_hydria",
+    "bashing": 1,
+    "material": "demon_chitin",
+    "symbol": ")",
+    "color": "light_gray",
```

Comment:
if it's made of demon chitin, i'd suggest red

Reply:
done